### ISSUE TITLE
bap may fail on macho binaries

### ISSUE THREAD

**Issue by dbrumley** [BinaryAnalysisPlatform/bap#414]
BAP is failing on my OSX El Capitan install:
```
$ bap /bin/ls 
LLVM ERROR: getSymbolValue unimplemented in MachOObjectFile
$ bap --version
0.9.9
$ ls /opt/local/bin/llvm-config-mp-3.*
/opt/local/bin/llvm-config-mp-3.4 /opt/local/bin/llvm-config-mp-3.5 /opt/local/bin/llvm-config-mp-3.7
```

**Comment by KennethAdamMiller** [comment]
I am on the latest mac and have had some trouble with this. My fix was to run llvm-config to select a particular llvm instance to default to. The latest llvm port package installs manage multiple versions with reconfigurable symlink trees like they should have ages ago. I think the command I used to fix this is:

sudo port select llvm mp-llvm-3.4

The other thing I've noticed about el capitan and bap is that locally built packages are failing with a no llvm backend error. But the bap-mc installed under opam resolves correctly... not sure what's up with that.

**Comment by ivg** [comment]
> The other thing I've noticed about el capitan and bap is that locally built packages are failing with a no llvm backend error. But the bap-mc installed under opam resolves correctly... not sure what's up with that.

I don't know what is "locally built package" for you, but this error suggests that `llvm` plugin is not loaded. To load `llvm` plugin you need to compile, install and load it. You can compile and install with opam, and load with `Plugins.load`.

**Comment by KennethAdamMiller** [comment]
These locally built binaries are those that used to work using the Bap.Std
interface but not a plugin. Thanks for the tip, I'll add the code to be
peeled away later.

**Comment by ivg** [comment]
fixed by #418, confirmed manually.
